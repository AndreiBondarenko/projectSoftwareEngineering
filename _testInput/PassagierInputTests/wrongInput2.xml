<!-- naam is missing -->
<PASSAGIER>
	<beginstation>A</beginstation>
	<eindstation>C</eindstation>
	<hoeveelheid>7</hoeveelheid>
</PASSAGIER>
<PASSAGIER>
	<naam>Groep2</naam>
	<beginstation>C</beginstation>
	<eindstation>B</eindstation>
	<hoeveelheid>4</hoeveelheid>
</PASSAGIER>
<PASSAGIER>
	<naam>Groep3</naam>
	<beginstation>B</beginstation>
	<eindstation>A</eindstation>
	<hoeveelheid>15</hoeveelheid>
</PASSAGIER>
<PASSAGIER>
	<naam>Groep4</naam>
	<beginstation>C</beginstation>
	<eindstation>A</eindstation>
	<hoeveelheid>10</hoeveelheid>
</PASSAGIER>
<PASSAGIER>
	<naam>Groep5</naam>
	<beginstation>A</beginstation>
	<eindstation>C</eindstation>
	<hoeveelheid>5</hoeveelheid>
</PASSAGIER>
