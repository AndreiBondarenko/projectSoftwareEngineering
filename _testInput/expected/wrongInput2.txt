XML PARTIAL IMPORT: Expected <STATION> ... </STATION> or <TRAM> ... </TRAM> and got <HALTE> ... </HALTE>.
